src/hpack/decoder.rs

```diff
@@ -474,8 +476,9 @@ impl Table {
         debug_assert!(size <= self.max_size);
 
         while self.size + size > self.max_size {
-            let last = self.entries.pop_back()
-                .expect("size of table != 0, but no headers left!");
+            let last = self.entries.pop_back().expect(
```

Comment:
Hm? Is this because the line is over the line length?

Reply:
yep

Reply:
fixed this by changing `fn_call_style`